apps/chat/src/chat/chat.service.ts

```diff
@@ -1,4 +1,25 @@
 import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { ChatRoom } from './entity/chat-room.entity';
 
 @Injectable()
-export class ChatService {}
+export class ChatService {
+  public constructor(@InjectModel(ChatRoom.name) private chatRoomModel: Model<ChatRoom>) {}
+
+  public async createRoom(roomName: string) {
+    const lastRoom = await this.chatRoomModel.findOne().sort({ channel_id: -1 }).exec();
```

Comment:
mongoose에 익숙한 개발자라면 이해하는데 큰 문제는 없을 것 같은데요~ 그렇다고 하더라도 되게 복잡한 코드를 읽을 때 -1과 같은 값을 보게 되면 어떻게 동작하는지 한번 생각하는 과정이 필요해서 상수로 다뤄줘도 괜찮을 것 같습니다!
```suggestion
    const lastRoom = await this.chatRoomModel.findOne().sort({ channel_id: MONGOOSE.SORT.DESC }).exec();
```

Reply:
[f151478](https://github.com/GrebbMe/grabbme-be/pull/35/commits/f151478e2788bf7adcfb958d2b59fd92200b86b8)

위 커밋에 반영하였습니다!